runtime/vm/BytecodeAction.hpp

```diff
@@ -47,16 +47,16 @@ typedef enum {
 	THROW_AIOB,
 	THROW_ARRAY_STORE,
 	THROW_DIVIDE_BY_ZERO,
-#if defined(J9VM_OPT_VALHALLA_VALUE_TYPES)
-	THROW_VALUE_TYPE_ILLEGAL_MONITOR_STATE,
-#endif /* defined(J9VM_OPT_VALHALLA_VALUE_TYPES) */
 	THROW_ILLEGAL_MONITOR_STATE,
 	THROW_INCOMPATIBLE_CLASS_CHANGE,
 	THROW_WRONG_METHOD_TYPE,
 	PERFORM_DLT,
 	RUN_METHOD_INTERPRETED,
 	RUN_JNI_NATIVE,
 	RUN_METHOD_COMPILED,
+#if defined(J9VM_OPT_VALHALLA_VALUE_TYPES)
```

Comment:
This still doesn't match the interpreter - the VT one is after the debug ones in the interpreter.